projects/external-components/src/lib/primeng-selectors/components/card-list/card-list.component.ts

```diff
@@ -61,7 +61,7 @@ export class CardListComponent extends CommonExternalComponent {
 
   private refreshCards() {
     this.loading = true;
-    this.customApiCall(this.fieldObj.customAttributes.apiConfig).subscribe((response: any) => {
+    this.customApiCall!(this.fieldObj.customAttributes.apiConfig).subscribe((response: any) => {
```

Comment:
Check the history of this line change. Wasnt it removed to fix some issue in mobile?